scripts/secretcheck.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# If the aws credentials file does not exist then just exit ok
+AWS_FILE=~/.aws/credentials
+if [ ! -f ${AWS_FILE} ]
+then
+  echo "No AWS Credentials to check"
+  exit 0
+fi
+
+ROOT=.
+AWS_SECRET=$(grep secret ${AWS_FILE} | cut -d '=' -f2)
```

Comment:
`credentials` file may have multiple profiles so this may produce multiple lines:
```
[default]
aws_secret_access_key = ...
aws_access_key_id = AKIA...
region = us-east-1

[sprint]
aws_secret_access_key = ...
aws_access_key_id = AKIA...
region = eu-west-2
```

Reply:
This should work now -as the bash array copes with space separations.